modules/admin_manual/pages/configuration/server/occ_commands/core_commands/_sharing_commands.adoc

```diff
@@ -21,3 +23,19 @@ So, to cleanup all orphaned remote storages, run it as follows:
 You can also set it up to run as xref:background-jobs-selector[a background job].
 
 NOTE: These commands are not available in xref:maintenance-commands[single-user (maintenance) mode].
+
+== Allow to Temporarily Ignore Invalid Federated Shares
+
+Currently, if a federated share is invalid or the api endpoint returns not found, the availability check would validate whether this is a problem with a server. If checks complete, that given share is removed. However, in some cases these checks might not be enough like in complex migrations of tightly federated setups. In that case, invalidation behaviour can be disabled using the below app setting. When set, instead of removing the share, a warning in the browser is displayed.
```

Comment:
```suggestion
Currently, if a federated share is invalid or the API endpoint returns a "not found", the availability check tests whether this is a problem with a server. If checks complete, that given share is removed. However, in some cases these checks might not be enough like in complex migrations of tightly federated setups. In that case, invalidation behavior can be disabled using the below app setting. When set, instead of removing the share, a warning is displayed in the browser.
```